cinema/views.py

```diff
@@ -167,6 +188,18 @@
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "date",
+                type={"type": "string", "items": {"type": "number"}},
```

Comment:
The OpenAPI parameter type for `date` is incorrectly specified as a list of numbers. It should be a string representing a date. Correct the type to `{"type": "string"}`.